roles/kubernetes/node/tasks/install.yml

```diff
@@ -39,7 +13,16 @@
     ]"
   tags: facts
 
-- name: install | Install kubelet launch script
-  template: src=kubelet-container.j2 dest="{{ bin_dir }}/kubelet" owner=kube mode=0755 backup=yes
-  notify: restart kubelet
+- include: install_rkt.yml
```

Comment:
Another item: If install_rkt.yml or install_docker.yml doesn't need `ssl_ca_dirs` var, can you move this include to the top? Putting them at the beginning or the end of the task file makes it more visible.